gen/js/component/motor/v1/motor_pb.js

```diff
@@ -3025,7 +3025,6 @@ proto.viam.component.motor.v1.Status.prototype.toObject = function(opt_includeIn
 proto.viam.component.motor.v1.Status.toObject = function(includeInstance, msg) {
   var f, obj = {
     isPowered: jspb.Message.getBooleanFieldWithDefault(msg, 1, false),
-    positionReporting: jspb.Message.getBooleanFieldWithDefault(msg, 2, false),
```

Comment:
the order and gaps in field ids don't matter. what matters is whether we ever re-use 2 as an id in the future.
the right thing to do is to reserve field id 2 so that it is never used in the future, thus preserving backward compatibility. in this case i believe it's safe to risk re-using it since 1. we're in an unstable beta state 2. the likelihood of us adding another status field in the near future is low.